EasyPost.NetFramework/EasyPost.NetFramework.csproj

```diff
@@ -24,6 +26,18 @@
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Test|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Test</OutputPath>
+    <DefineConstants>
+    </DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
+    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
+    <NoWarn>CS1591</NoWarn>
+  </PropertyGroup>
```

Comment:
Please double check these `csproj` files as it appears they still have a few things leftover from branching off the past branch which had changes since.

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)